azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/DeletedKeyItem.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+package com.microsoft.azure.keyvault.models;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The deleted key item containing the deleted key metadata and information
+ * about deletion.
+ */
+public class DeletedKeyItem extends KeyItem {
```

Comment:
This is a model from the updated spec we should remove it from the refactoring, and add back in a subsequent update.